services/soap/pom.xml

```diff
@@ -100,7 +111,6 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>${guavaVersion}</version>
```

Comment:
There are many dependencies in poms where the version attribute was removed. I guess, because the dependency is managed in the parent pom. This particular change is in a service. Services should have their own dependencies as they will be included inside the service artifact. (Probably a change that goes beyond this PR, but something we need to take in account)